python/ray/serve/backend_worker.py

```diff
@@ -71,7 +73,30 @@ async def __init__(self, backend_tag, replica_tag, init_args,
                 # This allows backends to define an async __init__ method
                 # (required for FastAPI backend definition).
                 _callable = backend.__new__(backend)
```

Comment:
do we need to `__new__` the class again upon retry? the state might be corrupted if we keep retrying broken inits?

Reply:
good point, we can always create new instance rather than re-using the same one. will address in later commits on this PR.